docs/troubleshooting.md

```diff
@@ -1 +1 @@
-{!TROUBLESHOOTING.md!}
+Check [troubleshooting.md](https://github.com/complytime/complyscribe/blob/main/TROUBLESHOOTING.md) 
```

Comment:
Should this be also uppercase to be consistent with "CONTRIBUTING.md"?

Reply:
Yes we should. Thanks. Updated.